recipes/nbgrader/build.sh

```diff
@@ -0,0 +1,4 @@
+"${PYTHON}" setup.py install --single-version-externally-managed --record=record.txt
+
+#"${PREFIX}/bin/jupyter-nbextension" install --sys-prefix --overwrite --py nbgrader
```

Comment:
@ocefpaf I am not sure if I can put this is the `meta.yaml` file as this line will also be part of the build script. Can the `build: script:` section in the `meta.yaml` file contain multiple commands?